x/emissions/module/rewards_calc.go

```diff
@@ -195,7 +248,7 @@ func emitRewards(ctx sdk.Context, am AppModule) error {
 		return err
 	}
 
-	return nil
+	return am.keeper.ResetNumInferencesInRewardEpoch(ctx)
```

Comment:
I don't love this line of code, i think it's weirdly confusing

I understand what you're doing here and it's logically equivalent, but for readability's sake i'd much prefer

```
err = am.keeper.ResetNumInferencesInRewardEpoch(ctx)
if err != nil {
    return err
}

return nil
```

Resetting the number of inferences in the reward epoch is a task the emitRewards function should do, not the return of that function, it's weird


it's a nitpick feel free to ignore me if you disagree